packages/terra-form/src/components/Input.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Input.scss';
+
+const propTypes = {
+  /**
+   * The defaultValue of the input field. Use this to create an uncontrolled input
+   */
+  defaultValue: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+  /**
+   * Name of the input
+   */
+  name: PropTypes.string,
+  /**
+   * Functional to be called when the input is changed
+   */
+  onChange: PropTypes.func,
+  /**
+   * Whether the input is required or not
+   */
+  required: PropTypes.bool,
+  /**
+   * The value of the input field. Use this to create a controlled input
+   */
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+};
+
+const defaultProps = {
+  defaultValue: undefined,
+  name: null,
+  onChange: () => {},
+  required: false,
+  value: undefined,
+};
+
+const Input = ({
+  defaultValue,
+  name,
+  onChange,
+  required,
+  value,
+  ...customProps
+}) => {
+  const additionalInputProps = Object.assign({}, customProps);
+
+  if (required) {
+    additionalInputProps['aria-required'] = 'true';
+  }
+
+  if (value) {
```

Comment:
That did the trick.

https://github.com/cerner/terra-core/pull/352/commits/9c897f626b62fba1f2b35c6d5d000b7d911d6030